torchvision/prototype/transforms/functional/_augment.py

```diff
@@ -15,7 +15,7 @@
         features.Image: K.erase_image,
     },
 )
-def erase(input: T, *, i: int, j: int, h: int, w: int, v: torch.Tensor, inplace: bool = False) -> T:
+def erase(input: T, *, i: int, j: int, h: int, w: int, v: torch.Tensor, inplace: bool) -> T:
```

Comment:
Unrelated to your API (more related to the previous implementation) but I recommend not to support in-place here since we don't support it in 90% of the transforms. I would do a scan in all places where inplace is support it and straight remove the functionality.